tools/tinygo-buildstatus/statusquo.go

```diff
@@ -181,7 +181,7 @@ var (
 		"smbios_transfer",
 		"smn",
 		"srvfiles",
-		"ssh",
+		//"ssh",
```

Comment:
Thanks, that will do for now. Once the tinygo networking efforts continue I will investigate what the underlying issue is here. Thanks for taking care of that.